pkg/webircgateway/transport_kiwiirc.go

```diff
@@ -59,7 +59,7 @@ func (t *TransportKiwiirc) makeChannel(chanID string, ws sockjs.Session) *Transp
 	_, remoteAddrPort, _ := net.SplitHostPort(ws.Request().RemoteAddr)
 	client.Tags["remote-port"] = remoteAddrPort
 
-	client.Log(2, "New kiwi channel on %s from %s %s", ws.Request().Host, client.RemoteAddr, client.RemoteHostname)
+	client.Log(2, "New kiwiirc client on %s from %s %s", ws.Request().Host, client.RemoteAddr, client.RemoteHostname)
```

Comment:
This is a new channel, not a new client. A client (connection) can have many channels. This is how we duplex multiple IRC connections through the same websocket